### PR TITLE
User centric secrets

### PR DESCRIPTION
**Pull request by ryansch**
I'm tired of sticking secrets in every project and then putting that file in .gitignore and .dockerignore.

Bonus: gnupg server hated me again so I finally got rid of gosu and dumb-init

### CODE REVIEW

File: lib/shipitron.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 module Shipitron
   CONFIG_FILE = 'shipitron/config.yml'.freeze
-  SECRETS_FILE = 'shipitron/secrets.yml'.freeze
+  SECRETS_FILE = '~/.config/shipitron/secrets.yml'.freeze
```
Comment (morganick) [inline lib/shipitron.rb:9]: So we're moving this into our home dir?

File: scripts/docker-entrypoint.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ else
 fi
 
 if ${BINARY} help "$1" 2>&1 | grep -q "shipitron $1"; then
-  set -- gosu shipitron ${BINARY} "$@"
+  set -- su-exec shipitron ${BINARY} "$@"
```
Comment (morganick) [inline scripts/docker-entrypoint.sh:11]: So we're making the switch to `su-exec` eh?

Comment (ryansch) [reply scripts/docker-entrypoint.sh:11]: Yep.

File: lib/shipitron.rb (hunk 1 above)
Comment (ryansch) [reply lib/shipitron.rb:9]: Yep.

**Comment by ryansch** [comment]
@morganick This branch can trigger shipitron deploys.

**Review by morganick — APPROVED** [review]

**Pull request merged**